feature-libs/organization/account-summary/components/account-summary/details/document/account-summary-document.module.ts

```diff
@@ -0,0 +1,23 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { I18nModule, provideDefaultConfig, UrlModule } from '@spartacus/core';
+import { IconModule, ListNavigationModule } from '@spartacus/storefront';
+import { AccountSummaryDocumentComponent } from './account-summary-document.component';
+import { accountSummaryDocumentCmsConfig } from './account-summary-document.config';
+import { AccountSummaryDocumentFilterModule } from './filter';
+
+@NgModule({
+  declarations: [AccountSummaryDocumentComponent],
+  imports: [
+    AccountSummaryDocumentFilterModule,
+    CommonModule,
+    RouterModule,
+    I18nModule,
+    ListNavigationModule,
+    UrlModule,
+    IconModule,
```

Comment:
Do we need `RouterModule`, `UrlModule` and `ListNavigationModule`?

Reply:
RouterModule, UrlModule removed, not used
ListNavigationModule replaced by SortingModule, PaginationModule, (only these 2 are used)